query-node/schemas/contentNft.graphql

```diff
@@ -263,3 +264,17 @@ type Bid @entity {
 }
 
 # TODO entity for (cancelable) offers; will be needed to see history of offers
+
+type CuratorAgentPermissions @entity {
```

Comment:
Can't this entity be named just `Curator`, similar to how `Collaborator` is named. Also in [Curator](https://github.com/Joystream/joystream/pull/4150/files#diff-ddf70c18d6730ebb157f4fa1dc7839a954e2807eb69de6ab907903a10e2ca9cdR44) the field `curatorGroups` seems confusing as its a list of `CuratorAgentPermissions`. Is it possible to remove [this](https://github.com/Joystream/joystream/pull/4150/files#diff-ddf70c18d6730ebb157f4fa1dc7839a954e2807eb69de6ab907903a10e2ca9cdR44) type, and use `CuratorAgentPermissions` renamed as `Curator` instead

Reply:
I don't think that's possible because Curator can be in multiple curator groups and can be a ContentActor as well.